src/repository/resources/ipres.rs

```diff
@@ -1709,12 +1709,128 @@ impl AddressFamily {
 }
 
 
+//------------ Ipv4Block -----------------------------------------------------
+
+/// Contains an IPv4 address block.
+///
+/// This type is a thin wrapper around [`IpBlock`] ensuring that this is an
```

Comment:
I'm not sure that referring to this as a "wrapper" is the right thing to do as you don't wrap the "encode" or "take_opt_xxx" functions.